internal/dev_server/sdk/routes.go

```diff
@@ -21,6 +21,9 @@ func BindRoutes(router *mux.Router) {
 
 	router.Handle("/all", GetProjectKeyFromAuthorizationHeader(http.HandlerFunc(StreamServerAllPayload)))
 
+	router.PathPrefix("/sdk/flags/{flagKey}").
```

Comment:
The `GetServerFlags` method already has a branch of logic that handles the individual flag request if the `flagKey` variable is set. However, we never defined that route so the variable would in fact get set.